src/core/AutoRest.Core/Validation/ValidationCategory.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace AutoRest.Core.Validation
+{
+    [Flags]
+    public enum ValidationCategory
```

Comment:
Super Cool bitwise 👍 

Minor: Most likely you don't need `System.Collections.Generic`, `System.Linq` & `System.Threading.Tasks`